OpenCombine.py

```diff
@@ -0,0 +1,22 @@
+# To use `OpenCombine.py`, figure out its full path.
+# Let's say the full path is `~/projects/OpenCombine/OpenCombine.py`.
+# Then add the following statement to your `~/.lldbinit` file:
+#
+#     command script import ~/projects/OpenCombine/OpenCombine.py
+
+import lldb
+
+# Show a Demand as either `max(N)` or `unlimited`.
+def summary_Demand(sb_value, internal_dict):
+    child = sb_value.GetChildAtIndex(0)
+    if not child.IsValid():
+        return 'failed to get child of Demand'
+    number = child.GetValueAsUnsigned()
+    size = child.GetByteSize()
+    if number == 2**(8 * size - 1):
```

Comment:
Let's make it clear what's going on:
```suggestion
    if number == 2**(8 * size - 1): # UInt(Int.max) + 1
```

Reply:
I've clarified the code in the new patch.